cidnfstack.py

```diff
@@ -641,11 +651,15 @@ def _start_commandline():  # pylint: disable=R0912,R0915
         description='Build RPMs of a project from the checkout in the current '
                     'working directory in Copr.')
     buildparser.add_argument(
-        '-ddd', '--dnf_docker_test', help='start the test of DNF command interface after copr build',
+        '-ddt', '--dnf_docker_test', help='start the test of DNF command interface after copr build',
```

Comment:
I will correct it, but to have a option -ddt it is very tempting if you think about.